File: FC.App/Scripts/Shared/ViewModels/FestivalDetailScope.ts
import Model = FC.Shared.Models;
module FC.Shared.ViewModels {
    export class FestivalDetailScope {
        public FestivalDetails: Model.UFestival; 
        public Artists: Array<Model.UArtist>; 
        public Genres: Array<Model.UGenre>; 
        public News:  Array<Model.UNews>;
    }
}

File: FC.Shared/ViewModels/Festival/FestivalDetailScope.cs
using FC.Shared.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FC.Shared.ViewModels.Festival
{
    public class FestivalDetailScope
    { 
        public UFestival FestivalDetails { get; set; }
        public List<UArtist> Artists { get; set; }
        public List<UGenre> Genres { get; set; }
        public List<UNews> News { get; set; }
    }
}
